src/ansys/edb/core/session.py

```diff
@@ -553,3 +551,23 @@ def get_variable_server_stub():
     VariableServerServiceStub
     """
     return StubAccessor(StubType.variable_server).__get__()
+
+
+def __ensure_session(ansys_em_root, port_num, ip_address):
```

Comment:
@chenchienjacklin can you change this to one underscore? two underscore is reserved for name mangling